lib/config/ui_config.dart

```diff
@@ -89,6 +89,7 @@ abstract class WideTileCardConfig {
 }
 
 abstract class DetailViewsConfig {
+  static const heightImage = 254.0;
   static const double spacerHeight = 16;
```

Comment:
please specify the type of `heightImage`, imo better to rename it to: `imageHeight` (just to follow convention)

Reply:
Changed -  i did it as it was done for heightBig heightSmall etc